wikipedia-dark.user.css

```diff
@@ -4192,6 +4194,26 @@ regexp("https?:\/\/([\w\-]{2,}\.m\.(wiki(pedia|books|news|quote|source|versity|v
   tr[bgcolor], td[bgcolor] {
     background-color: var(--gray-28);
   }
+  /* popups fixes */
+  .mwe-popups .mwe-popups-container {
+    background-color: var(--gray-2) !important;
+  }
+  .mwe-popups .mwe-popups-container .mwe-popups-discreet {
```

Comment:
Please target the image `.mwe-popups-thumbnail` instead of the wrapper, when you target the wrapper, it sometimes leaves a large area of white background exposed.

Reply:
Let me check that one. The purpose of this code is to apply white background on images so svgs would still be readable